src/codegen/compile.ml

```diff
@@ -5756,6 +5755,10 @@ module Serialization = MakeSerialization(BumpStream)
 
 module Stabilization = struct
 
+  let extend64 code = code ^^ G.i (Convert (Wasm.Values.I64 I64Op.ExtendUI32))
```

Comment:
My style so far was to pass `code` only when the argument can’t simply be the top element of the stack. Higher order stuff is alwasy harder to understand when called. So I’d advise against it.

You maybe could add `compile_extend64` to our list of `compile_…` functions. (And maybe they need a shorter naming convention).

Reply:
My hope is that we'll be able to get rid of these conversions when gradually more variables/data trust to 64 bits. This `extend64` is well-confined here and   will be hopefully eliminated soon in one way or the other.